scipy/stats/tests/test_distributions.py

```diff
@@ -2381,6 +2381,34 @@ def test_logcdf(self):
                         stats.norm.logsf(np.log(x2-mu)/sigma))
 
 
+# Columns are x, a, b, cdf, sf.
+# The values of cdf and sf were computed using mpmath.
+_beta_cdf_sf = np.array([
+    [1e-20, 3.0, 0.5, 3.1249999999999994e-61, 1.0],
+    [1e-20, 0.5, 3.0, 1.875e-10, 0.9999999998125],
+    [0.999, 2, 10, 1.0, 1.0990000000000097e-29],
+    [0.9999999999999994, 3.0, 0.5, 0.9999999558234914, 4.417650858175538e-08],
```

Comment:
```suggestion
    [0.9999999999999994, 3.0, 0.5, 0.9999999540720673, 4.5927932677184570e-08],
```
Did you use mpf('0.9999999999999994') for the input? What you have is what I get when I enter the float as input.